docker-compose.override.yml

```diff
@@ -39,6 +39,8 @@ services:
       args:
         PARENT_IMAGE: ${REGISTRY}demo/${COMPOSE_PROJECT_NAME}-nodejs:${SITECORE_VERSION}-${NANOSERVER_VERSION}-${DEMO_VERSION}
     mem_limit: 2GB
+    ports:
+      - "8099:3000"
```

Comment:
Is this port needed for development purposes in general? If it is, could you please add some documentation on how to use it?